client/app/bundles/server-components/components/CommentsFeed.jsx

```
@@ -0,0 +1,85 @@
+import { Marked } from 'marked';
+import { gfmHeadingId } from 'marked-gfm-heading-id';
+import sanitizeHtml from 'sanitize-html';
+import _ from 'lodash';
+import TogglePanel from './TogglePanel';
+
+const marked = new Marked();
+marked.use(gfmHeadingId());
+
+async function CommentsFeed() {
+  // Simulate network latency to demonstrate streaming
+  await new Promise((resolve) => setTimeout(resolve, 800));
```

ESLint

[error] Return values from promise executor functions cannot be read.

(no-promise-executor-return)